src/browsers/schema/content/homepage.md

```diff
@@ -0,0 +1,9 @@
+Examples - Gene: [SETD1A](link), Variant: [16-30992057-CAG-C](link)
```

Comment:
The SCHEMA browser doesn't support direct links to variants, only genes.

Also, the URL for the SETD1A gene page can be added: https://schema.broadinstitute.org/gene/ENSG00000099381

Reply:
OK fixed the linking to the SETD1A page.